ClientResources/ViewMode/RecaptchaElementBlock.js

```diff
@@ -46,11 +46,13 @@ function initRecaptchaElements() {
         $(".Form__Element.FormRecaptcha").each(function (index, element) {
             var $element = $(element),
                 $widgetContainer = $(".g-recaptcha", $element),
-                siteKey = $element.data("epiforms-sitekey");
+                siteKey = $element.data("epiforms-sitekey"),
+                initialized = $element.data("epiforms-initialized");
```

Comment:
you check for "epiforms-initialized"
but you set to "epiforms-recaptcha-initialized"

is it mistake?